$NetBSD: patch-.._package.xml,v 1.3 2018/01/10 10:05:17 jdolecek Exp $

Update md5 sum. Remove once the config.m4 patch is merged, reported
upstream as #75791.

--- ../package.xml.orig	2018-01-02 14:08:16.000000000 +0000
+++ ../package.xml
@@ -121,7 +121,7 @@
    <file md5sum="9ef781739883eec8bb8743df9083152b" name="pgsql_s_lock.h" role="src" />
    <file md5sum="c95208eaf18327335d822da86eaa5bcf" name="pgsql_s_lock.c" role="src" />
    <file md5sum="610c03c9de0306986a66f3c6d2d97b9c" name="apc_serializer.h" role="src" />
-   <file md5sum="13d18fe32d6b8ed48f276349e54355aa" name="config.m4" role="src" />
+   <file md5sum="7c0ecf1beafb4f5a61aa6ba34498beb7" name="config.m4" role="src" />
    <file md5sum="e76c6931b54dc5fb3f55b20cd53aea04" name="config.w32" role="src" />
    <file md5sum="d6c836c279b1fbabf69ada700dd5cdfc" name="INSTALL" role="doc" />
    <file md5sum="52dd90569008fee5bcdbb22d945b1108" name="LICENSE" role="doc" />
